rust/private/rust.bzl

```diff
@@ -630,8 +638,12 @@ _rust_test_attrs = {
         mandatory = False,
         doc = dedent("""\
             Specifies additional environment variables to set when the test is executed by bazel test.
-            Values are subject to `$(execpath)` and
+            Values are subject to `$(rootpath)`, `$(execpath)`, location, and
             ["Make variable"](https://docs.bazel.build/versions/master/be/make-variables.html) substitution.
+
+            Execpath returns absolute path, and in order to be able to construct the absolute path we
```

Comment:
Hmmm, on second though, I feel like this is something that should be an incompatibility flag. I don't think it's too difficult to have users join `pwd` and their value themselves. I'm definitely interested in continuing to use `execpath` but also want to have debugger support. Maybe `rules_rust` can provide some utility crate for expanding `execpath` into absolute paths? Not saying that has to be done in this PR just trying to figure out if there's a path to completely abolish the launcher and maintain the ability to use absolute paths.

Reply:
I think you're saying that you support my followup PR to remove the launcher alltogether using an incompatible flag, and that this PR is good to go.

Or did you want to say that this PR needs an incompatible flag?

Reply:
I was thinking that there'd be no changes in behavior outside that flag. So unless it's flipped you'd still use the launcher. But perhaps that's more restrictive than necessary. But I wonder if anyone would run into issues in this change or be confused why two seemingly similar tests might behave differently.

Reply:
> Or did you mean that this PR needs an incompatible flag?

This is my question and I think it does but hard to really make up my mind since my attention is currently divided 😅

Reply:
My experience is that people encounter issues and confusion when using the launcher. Except really unlikely situations I don't think we would be breaking people by not using the launcher. Of course if we stop producing absolute paths that can really break people and we should guard it with an incompatible change.

Reply:
If you think this PR warrants the incompatible flag, I d make it a different one from the one that removes absolute paths, since they both have different fallout impact and migration stories.

Reply:
I can accept this as a middle ground but hope that we can work quickly together to get a feature flag up. As mentioned in another comment. I think the solution for absolute paths would actually be in Bazel proper and not the rules here.